packages/slas-react-sdk/src/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { createContext, useContext } from "react";
+import { createStore, useStore } from "zustand";
+import { persist } from "zustand/middleware";
+import { helpers, ShopperLogin } from "commerce-sdk-isomorphic";
+import { onClient, getAppOrigin } from "./util";
+import { isExpired } from "./jwt";
+import { MemoryStorage } from "./storage";
+
+enum AuthTypes {
+  REGISTERED = "registered",
+  GUEST = "guest",
+}
+
+interface State {
+  isInitialized: boolean;
+  isInitializing: boolean;
+  isAuthenticated: boolean;
+  authType: AuthTypes;
+  customerId: string;
+  accessToken: string;
+  refreshToken: string;
+  _config: CommerceAPIConfig;
+  login: (username: string, password: string) => void;
+  register: (username: string, password: string) => void;
+  loginAsGuest: () => void;
+  logout: () => void;
+  _initialize: () => void;
+  _refreshAccessToken: () => void;
+  _handleShopperLoginTokenResponse: (res: helpers.TokenResponse) => void;
+}
+
+interface CommerceAPIConfig {
+  clientId: string;
+  organizationId: string;
+  shortCode: string;
+  siteId: string;
+}
+
+interface ProviderProps extends CommerceAPIConfig {
+  proxy?: string;
+  debug?: boolean;
+  children: React.ReactNode;
+}
```

Comment:
It would be nice to make use of the type `ShopperLoginParameters` in the CommerceSDKIsomorphic types.